Multifamily masonry walls

JLReyna opened the pull request:
Addresses #368.

## Pull Request Description

Adding in masonry walls via "Geometry Wall Type" tsv and adding a new dependency in "Insulation Wall". Mimics work done by Maharshi for project_singlefamily. Also created a new tsv_maker for RECS 2009

## Checklist

Not all may apply:

- [ ] ~Unit tests have been added or updated~
- [X] All rake tasks have been run, and pass
- [x] PAT project measures/outputs have been updated
- [ ] ~Documentation has been modified appropriately~
- [ ] ~Any new options are added to `project_testing`~
- [ ] ~`project_testing` runs without any failures~
- [x] No unexpected regression test changes
- [x] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [X] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

ejhw commented on project_multifamily_beta/housing_characteristics/Geometry Wall Type.tsv:
```diff
@@ -0,0 +1,91 @@
+CR11	1980s	0.702261	0.297739	232.000000	1781272.250000
+CR11	1990s	0.704148	0.295852	177.000000	1317830.875000
+CR11	2000s	0.832828	0.167172	129.000000	948427.625000
+CR11	2010s	0.832828	0.167172	0.000000	0.000000
```
I'm a little bit suspicious of these high masonry numbers in CA. Presumably because they have a lot of stucco? Let's flag this somehow to revisit someday, perhaps by purchasing this:  "https://www.homeinnovation.com/trends_and_reports/data/new_construction/structural_systems_for_walls"

JLReyna replied:
It is quite high. Do you want to create an issue around it?

ejhw commented on project_multifamily_beta/housing_characteristics/Geometry Wall Type.tsv:
```diff
@@ -0,0 +1,91 @@
+CR11	1990s	0.704148	0.295852	177.000000	1317830.875000
+CR11	2000s	0.832828	0.167172	129.000000	948427.625000
+CR11	2010s	0.832828	0.167172	0.000000	0.000000
+CR11	<1940	0.659331	0.340669	133.000000	1036533.437500
```
Add "Created by RECS tsv maker using 2009 RECS data (WALLTYPE Masonry = Brick, Stucco, Stone, or Concrete/Concrete Block)" (or similar)

JLReyna replied:
Leaving out, per Tony's request (see note above)

ejhw commented on project_multifamily_beta/housing_characteristics/Insulation Wall.tsv:
```diff
@@ -1,91 +1,181 @@
+CR08	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.600000	0.220000	0.180000	0.000000
+CR09	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.670000	0.150000	0.170000	0.000000
+CR10	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.410000	0.200000	0.390000	0.000000
+CR11	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.470000	0.020000	0.510000	0.000000
```
Add "Created manually using data from Home Innovation Research Labs Builder Practices Reports (1990s and 2000s) and NAHB Builder Practices Reports (1980s). Expert elicitation used for all other years." or similar

JLReyna replied:
How soon are you wanting to do a run on masonry walls? I can add the tag, but I'm not sure which format to use or when Tony will have the update made (the difference is that he wants the tag as a comment with a hashtag). 

As a side note, I really dislike having the tag directly in the TSV. It's just a pain to do calculations with the TSV when you have to strip out the tag first. I'd rather have a central reference file that listed all the sources and automatically populated updates when tsv_maker.py was run.

JLReyna commented on data/recs/2009/tsv_maker_2009.py:
```diff
@@ -0,0 +1,292 @@
+
+    def export_and_tag(self, df, filepath, project):
+        """
+        Add bottom-left script and source tag to dataframe (for non testing projects). Save dataframe to tsv file.
```
Tony requested I leave it out because he is changing the format, and if I do it the new way it will fail integrity checks now, and if I do it the old way it will fail integrity checks later.

ejhw replied:
Ok 👍

JLReyna commented on data/recs/2009/parameter_option_maps.py:
```diff
@@ -0,0 +1,117 @@
+                                                         26: 'CR11',
+                                                         27: 'CR06',
+                                                         })
+    # Split out Kentucky and put in 8:
```
AK and HI impact all TSVs, so I remove them here so we're not accidentally doing calculations on them. I think this is an error in the RECS 2015 version of tsv_maker that they're not split out before the bedroom and occupant calculations. Kentucky is a Location Region specific modification, not for all TSVs, so I put it in parameter_option_maps.py

JLReyna commented:
I just updated 2015 RECS tsv_maker to also remove Alaska and Hawaii. We don't have reportable domains in 2015, but it was easy to identify them using IECC climate zone and census division. I also updated the names of the counts and weights to the new names that Tony wants to use.

joseph-robertson commented:
I still need to update PAT project outputs, but then this should be ready to go.
